apps/widget-configurator/src/app/configurator/hooks/useWidgetParamsAndSettings.ts

```diff
@@ -11,8 +11,8 @@ const getBaseUrl = (): string => {
   if (isLocalHost) return 'http://localhost:3000'
   if (isDev) return 'https://dev.swap.cow.fi/'
   if (isVercel) {
-    const prKey = window.location.hostname.replace('widget-configurator-git-', '').replace('-cowswap.vercel.app', '')
-    return `https://swap-dev-git-${prKey}-cowswap.vercel.app`
+    // const prKey = window.location.hostname.replace('widget-configurator-git-', '').replace('-cowswap.vercel.app', '')
+    return `https://swap-dev-git-fix-4289favorite-tooltip-for-widget-cowswap.vercel.app`
```

Comment:
Just for testing.

This PR's URL breaks the pattern making the widget not work.

Revert before merging.

Reply:
Reverted